lib/index.js

```diff
@@ -228,15 +229,15 @@ module.exports = {
   /**
    * Gets all of the processes running
    * 
-   * Warnings:
-   * - This will not work in a Windows machine
-   * 
    * @param {number} pid The number of processes to list
-   * @works Linux, MacOS
+   * @works Windows, Linux, MacOS
    * @returns An array of all the processes running by the `pid` argument
    */
-  getProcesses(pid = 10) {
-    if (this.getPlatform() === 'Windows') throw new SyntaxError('The "getProcesses" function requires to be in a Unix-subsystem');
+  async getProcesses(pid = 10) {
+    if (this.getPlatform() === 'Windows') {
+      const processes = await psList();
+      return processes.slice(0, 10);
```

Comment:
Should it slice how many the `pid` argument should give?

Reply:
Forgot about this